manifests/state-client-pod.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   serviceAccountName: nmstate-state-controller
   containers:
   - name: state-client
-    image: yuvalif/k8s-node-net-conf-client
+    image: yuvalif/kubernetes-nmstate-client
```

Comment:
this image does not exists on hub.docker.com under yuvalif

Reply:
Right, good point.

Reply:
@yuvalif should be fixed now, could you please take another look?